### PR TITLE
Added procedure card to the patient dashboard

### LINKED ISSUE #4871: View procedures in Patient Dashboard
**Is your feature request related to a problem? Please describe.**
We are recording procedures suggested for a patient in the consultation form (https://care.coronasafe.in/facility/e6b40cf4-4196-4e7d-83f0-b0bf5c38458a/patient/b80bbd73-8d8b-47ba-8a71-12a2bbfd65b2/consultation) However, we are not viewing the information anywhere.

**Describe the solution you'd like**
Add a card in the Patient Dashboard below "General Instructions" to show additional details about each of the procedures suggested. Keep the design consistent with the "prescription" and "General Instructions" cards.

### PR DESCRIPTION
**Pull request by patelaryan7751**
### <samp>🤖 Generated by Copilot at 92baa18</samp>

This pull request adds a feature to show the procedure details of a consultation in the `ConsultationDetails` component. It also improves the code readability and consistency in `src/Components/Facility/ConsultationDetails.tsx`.

## Proposed Changes

- Fixes #4871 
Added procedure card to the patient dashboard
image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### <samp>🤖 Generated by Copilot at 92baa18</samp>

*  Add a new feature to display the procedure details of the consultation ([link](https://github.com/coronasafe/care_fe/pull/5464/files?diff=unified&w=0#diff-f6f40e1b249471f1821a43cd90bfdce470f563c22b22272a154f9bd7a2de599cR915-R955))
* Improve the readability and formatting of the component by adding an empty line of code ([link](https://github.com/coronasafe/care_fe/pull/5464/files?diff=unified&w=0#diff-f6f40e1b249471f1821a43cd90bfdce470f563c22b22272a154f9bd7a2de599cR900))
* Remove an unnecessary empty line of code that does not affect the functionality of the component ([link](https://github.com/coronasafe/care_fe/pull/5464/files?diff=unified&w=0#diff-f6f40e1b249471f1821a43cd90bfdce470f563c22b22272a154f9bd7a2de599cL15))

### CODE REVIEW

**Comment by nihal467** [comment]
@patelaryan7751 can you remove the procedure from the medicine section and mark it for testing

**Comment by patelaryan7751** [comment]
Hey @nihal467 i have removed the procedure from medicine tab

File: package-lock.json
Comment (rithviknishad) [inline package-lock.json:1]: Did you accidentally commit this file?

Comment (rithviknishad) [inline package-lock.json:1]: Did you accidentally delete this file now? 😅

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (patelaryan7751) [reply package-lock.json:1]: yeahh it got commited i deleted it.

Comment (rithviknishad) [reply package-lock.json:1]: you do realize that upon merging this, this file would be deleted even from develop branch right? 😅

Comment (patelaryan7751) [reply package-lock.json:1]: yeah i would be giving a PR in other branch my package.lock.json got updated

**Pull request closed without merging**